components/apimgt/org.wso2.carbon.apimgt.core/src/main/java/org/wso2/carbon/apimgt/core/configuration/models/DataPublisherConfigurations.java

```diff
@@ -26,7 +26,7 @@
 public class DataPublisherConfigurations {
 
     @Element(description = "Reciever URL")
-    private String receiverURL = "tcp://localhost:7611";
+    private String receiverURL = "tcp://localhost:9612";
```

Comment:
Hi, yes. in C5 configuration model, we have to define a default values. So that we have to hardcode in this way